view.go

```diff
@@ -235,8 +235,7 @@ func (v *View) setRune(x, y int, ch rune, fgColor, bgColor Attribute) error {
 //   y < total lines && y > 0
 //   (x < view width || x < y's line width) && x > 0
 func (v *View) SetCursor(x, y int) error {
-	maxX, _ := v.Size()
-	if x < 0 || y < 0 || y >= len(v.lines) || (len(v.lines[y]) >= x && x >= maxX) {
+	if x < 0 || y < 0 || (y >= len(v.lines) && y != 0) || (x > 0 && (len(v.lines) == 0 || len(v.lines[y]) >= x)) {
```

Comment:
I think there is typo on the comparison. Was testing it on my app, which works with 0 but not when adding text and then setting cursor after that text (like autocompletion). I think the last part should be `< x`. I created suggestion.     
I didn't go with `<= x` because that wouldn't allow to to set cursor after text if you use like this:
```go
fmt.Fprint(v, "text")
v.SetCursor(len("text"),0)
```
Hope it makes sense :) 

```suggestion
	if x < 0 || y < 0 || (y >= len(v.lines) && y != 0) || (x > 0 && (len(v.lines) == 0 || len(v.lines[y]) < x)) {
```

Reply:
Thanks that's indeed incorrect have fixed it.